scripts/genGoverningChainTargetedProposalArgs.ts

```diff
@@ -0,0 +1,60 @@
+import { RoundTripProposalCreator, L1GovConfig, UpgradeConfig } from "../src-ts/proposalCreator";
+import { JsonRpcProvider } from "@ethersproject/providers";
+import { DeployedContracts } from "../src-ts/types";
+import fs from "fs";
+
+import dotenv from "dotenv";
+import { BigNumber, utils } from "ethers";
+
+const goerliDeployedContracts = JSON.parse(
+  fs.readFileSync("./files/goerli/deployedContracts.json").toString()
+) as DeployedContracts;
+const mainnetDeployedContracts = JSON.parse(
+  fs.readFileSync("./files/goerli/deployedContracts.json").toString()
+) as DeployedContracts;
```

Comment:
would be good to have a utility function that handles the import and typecast (including typeguards)